torch/onnx/symbolic_opset13.py

```diff
@@ -163,13 +163,21 @@ def reduce(g, *args, **kwargs):
         @parse_args("v", "none")
         def reduce_nodim(g, self, dtype):
             if dtype.node().kind() != "prim::Constant":
-                return _unimplemented(name, "dtype")
+                if dtype.node().kind() == "onnx::Constant":
```

Comment:
I'd restructure this to avoid a level of nesting:

```python
if kind == "onnx::Constant":
  ...
elif kind != "prim::Constant":
  return _unimplemented()
```